gemini/evaluation/define_your_own_metrics_rapid_evaluation_sdk.ipynb

```diff
@@ -1,913 +1,900 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    import plotly.graph_objects as go
</pre><p>This one can go to "General"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>This is fix by lint to make the import order is correct. Remove #General and #Main comment</p>